### ISSUE TITLE
Errors are reported twice

### ISSUE THREAD

**Issue by minooo** [eslint/eslint#9239]
![image](https://user-images.githubusercontent.com/9410091/30091796-2073edc8-92ee-11e7-8edb-d690552c9d67.png)

* **ESLint Version: ^4.3.0**
* **Node Version: 8.4**
* **npm Version: 5.3**

**if the file endswith '.jsx', a error is reported once, it is ok, but when '.js', the same error is reported twice!**

```js
module.exports = {
    root: true,
    env: {
        es6: true,
        browser: true,
        node: true,
        mocha: true,
    },

    // babel-eslint 支持实验性语法，比如在class中定义箭头函数
    parser: 'babel-eslint',

    // https://eslint.org/docs/user-guide/configuring#specifying-parser-options
    parserOptions: {
        ecmaVersion: 6,
        sourceType: 'module',
        ecmaFeatures: {
            globalReturn: true,
            impliedStrict: true,
            experimentalObjectRestSpread: true
        }
    },

    // https://eslint.org/docs/user-guide/configuring#specifying-parser-options
    extends: [
        'airbnb',
        'plugin:react/recommended',
        'prettier',
        'prettier/react'
    ],
    plugins: [
        'react',
        'prettier',
        'babel',
        'import'
    ],

    // https://eslint.org/docs/rules/
    rules: {
        indent: [
            'error',
            4
        ],
        'linebreak-style': [
            'error',
            'unix'
        ],
        quotes: [
            'error',
            'single'
        ],
        semi: [
            'error',
            'never'
        ],
        'no-return-assign': ['error', 'except-parens'],
        // 一个函数的复杂性不超过 10，所有分支、循环、回调加在一起，在一个函数里不超过 10 个
        'complexity': [
            2,
            10
        ],
        // 一个函数的嵌套不能超过 4 层，多个 for 循环，深层的 if-else，都是罪恶之源
        'max-depth': [
            2,
            4
        ],
        // 一个函数最多有 3 层 callback，使用 async/await
        'max-nested-callbacks': [
            2,
            3
        ],
        // 一个函数最多 5 个参数。参数太多的函数，意味着函数功能过于复杂，请拆分
        'max-params': [
            2,
            5
        ],
        // 一个函数最多有 50 行代码，如果超过了，请拆分之，或者精简之
        'max-statements': [
            2,
            50
        ],

        'react/jsx-filename-extension': [1, { "extensions": [".js", ".jsx"] }],

        'import/no-unresolved': 0,
        'import/extensions': 0,

        'prettier/prettier': [
            'error',
            {
                // api参考 https://github.com/prettier/prettier#options
                singleQuote: true,
                trailingComma: 'all',
                printWidth: 100,
                tabWidth: 4,
                semi: false,
                bracketSpacing: true,
                jsxBracketSameLine: true,
                parser: 'flow'
            }
        ],
    }
}

```

**Comment by eslintbot** [comment]
Hi @minooo, thanks for the issue. It looks like there's not enough information for us to know how to help you.

If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

Requesting a rule change? Please see [Proposing a Rule Change](http://eslint.org/docs/developer-guide/contributing/rule-changes) for instructions.

If it's something else, please just provide as much additional information as possible. Thanks!

**Comment by minooo** [comment]
@not-an-aardvark sorry, but my question is ready now

**Comment by platinumazure** [comment]
Does the double reporting happen when running ESLint from the command line?

What happens if you comment out seemingly unrelated plugins from your configuration?

If there is an issue, it will be much easier for us to fix it if we have a smaller/simpler example to work with. Your example configuration is fairly large :smile: Please try removing rules, plugins, config extensions, etc. until you have something much smaller that still reproduces the problem. Thanks!

**Comment by kaicataldo** [comment]
Closing this issue since we haven't heard anything back for quite some time. Feel free to create a new issue if this is something you would like to discuss further.
